### PR TITLE
Client meeting minutes for Dec. 2

### PR DESCRIPTION
**Pull request by vnajack**
I need to be somewhere at 4:30, so I'm uploading these without much review. Please let me know if any info is incorrect!

### CODE REVIEW

File: docs/meetings/client/2022_12_02.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- this week, they were working on their animator request flushed out
+- will send the word document outlining what frames are needed (mouths and poses), animator notes today
+- 22 different mouths, many may overlap with other teams
+- same characters, or easy colour changes
+- API is almost done, needs to be uploaded
+
+## Team C
+
+- does Kukarella have a virtual private server to test on?
+  - Erik: maybe? ask Group B what they did or try a cloud service for free trial, or just have local API working. It
```
Comment (LoganParker) [inline docs/meetings/client/2022_12_02.md]: sounded like basically a no, I'm under the impression they're interested in keeping their systems separate from ours until the end, and handling implementation themselves

File: docs/meetings/client/2022_12_02.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- The API for the demo API on the front page, is that fine for us to use?
+  - Erik: the demo is fine to use, they charge ~10-15 for 1 million characters
+- The free API demo... we can make 20-30 calls within a couple minutes - just wanted to check on blocks
+  - Erik: thank you for pointing that out, it was made a while ago so we'll check it out
+
+## To-do List
+
+- Animator may need feedback over the break, so please have at least 1 person per team available on Discord to check
+every couple days just in case
+- Determine scheduling for next semester for the client meeting
```
Comment (LoganParker) [inline docs/meetings/client/2022_12_02.md:60]: Jan 8. check in?

**Review by LoganParker — APPROVED** [review]
Thanks for taking notes! all looks great

**Review by edouarde1 — APPROVED** [review]
These look great! Thank you!!

**Review by mkuelker — APPROVED** [review]
Looks good!

File: docs/meetings/client/2022_12_02.md (hunk 2 above)
Comment (vnajack) [reply docs/meetings/client/2022_12_02.md:60]: Jan 8 sounds good! I'll post in Discord about this, too!

File: docs/meetings/client/2022_12_02.md (hunk 1 above)
Comment (vnajack) [reply docs/meetings/client/2022_12_02.md]: Yeah, I think it was more a no as well. I'll adjust

Comment (vnajack) [inline docs/meetings/client/2022_12_02.md]:
```suggestion
  - Erik: no. ask Group B what they did or try a cloud service for free trial, or just have local API working. It

```

**Pull request merged**